cloudformation/travis.template

```diff
@@ -60,6 +68,34 @@
                     "Ref": "MapboxTileCopyTestUser"
                 }
             }
+        },
+        "MapboxTileCopyKMSKey": {
+            "Type" : "AWS::KMS::Key",
+            "Properties" : {
+                "Description": "KMS key for mapbox-tile-copy test user",
+                "Enabled": true,
+                "KeyPolicy" : {
+                    "Version": "2012-10-17",
+                    "Id": "test-user-kms",
+                    "Statement": [{
+                        "Effect": "Allow",
+                        "Principal": {
+                            "AWS": {
+                                "Fn::Join" : [ "", [ "arn:aws:iam::", { "Ref" : "AWS::AccountId" }, ":root" ] ]
```

Comment:
Just for reference, `Fn::Sub` is a wonderful thing:

```json
"AWS": { "Fn::Sub": "arn:aws:iam::${AWS::AccountId}:root" }
```

Reply:
Nice! Done.